README.md

```
@@ -1,419 +1,332 @@
-![Circuit](Add your circuit diagram here)
-*Add caption explaining connections*
+https://drive.google.com/drive/folders/1xeYC5Fjy-IYdIMTfzBi2oceXlEMsTBLe?usp=sharing
 
-![Schematic](Add your schematic diagram here)
-*Add caption explaining the schematic*
 
-#### Build Photos
+#### System Architecture
 
-![Team](Add photo of your team here)
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)